plenum/server/node.py

```diff
@@ -836,7 +875,7 @@ def checkInstances(self) -> None:
         logger.debug("{} choosing to start election on the basis of count {} "
                      "and nodes {}".format(self, self.connectedNodeCount,
                                            self.nodestack.conns))
-        self._schedule(self.decidePrimaries)
+        # self._schedule(self.decidePrimaries)
```

Comment:
Why did we commented this? Will a primary be chosen for a new instance of the protocol?